content/guides/astro.md

```diff
@@ -17,11 +17,172 @@ comments: false
 draft: false
 ---
 
+{{< hextra/hero-subtitle >}}
+  Astro is an all-in-one web framework for building content-driven websites, that pioneers a new frontend architecture to reduce JavaScript overhead and complexity. Learn in this guide how to deploy an Astro site on Clever Cloud.
+{{< /hextra/hero-subtitle >}}
+
+## Overview
+
+Clever Cloud supportes deploying both fully static and on-demand rendered Astro projects. Regardless of your `output` mode ([pre-rendered or on-demand](/en/basics/rendering-modes/)), you can choose to deploy as a **static application** which runs using a post-build hook, or as a **Node.js** application, which works out-of-the-box with your `package.json`.
+
 If you need an example source code, get [Astrowind](https://github.com/onwidget/astrowind) (you'll need [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [Node.js](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs)):
 ```bash
 git clone https://github.com/onwidget/astrowind myStaticApp
 ```
 
+## Port and host
+
+Applications on Clever Cloud listen on port **8080**. If your project requires this configuration, set your port and host in Astro in one of two locations:
+
+{{< tabs items="package.json scripts,astro.config.mjs" >}}
+
+  {{< tab >}}
+
+   ```json {filename="package.json"}
+    "scripts": {
+      "dev": "astro dev",
+      "start": "astro dev",
+      "build": "astro check && astro build",
+      "preview": "astro preview --host 0.0.0.0 --port 8080",
+      "astro": "astro"
+    } 
+    ```
+  {{< /tab >}}
+
+  {{< tab >}}
+
+   ```javascript {filename="astro.config.mjs"}
+    import { defineConfig } from 'astro/config';
+
+    export default defineConfig({
+      server: {
+        port: 8080,
+        host: true
+      }
+    });
+    ```
+
+  {{< /tab >}}
+
+
+{{< /tabs >}}
+
+## Deploy Astro from the Console
```

Comment:
This kind of instructions are more a "How to use Console" guide than a "How to deploy this tech/framework on Clever Cloud" guide. It's why I usually detail only CLI instructions which show what's needed/important (and because the user can just copy/paste the code and see it deploying). We could refer to Console documentation for those needing instructions about it.

Reply:
It's a copy from the Astro guide, which of course will be more explicit to provide a glance of the Console. However, we need to keep in mind that people reading our guides will probably do so from some Google search. Imo, guides need to be as complete as possible, even if it means including beginner-like content, because it can be the only thing people will read from our doc, just to deploy a framework. The more step-by-step content, the better, since from a single page, they'll be able to deploy their site.

If we add too many reference on out guides page, the risk of people thinking "man, now I got to learn how the damn console works, oh and with is the CLI" increases. Guides are meant to be a one-shot page instructions from start to end, found on a search, complete enough to improve SEO with quality content and reduce the learning curve of Clever Cloud deployment. A single guide can replace many pages of this doc, and make someone competent enough to deploy without needing further instructions.

TL; DR: baby steps are staying

Reply:
>  (developers) can handle to use a node CLI

Let's not assume what our targeted audience can manage /will be willing to use or not. Both Console-focused and CLI-focused instructions should be able to fit into guides, and users will choose whichever option suits them better.

But I see your point. I propose that we follow the tunnel creation of the Console for the steps, without detailing it so much (it's a little bit over the top since it was made for Astro doc), and that we bring the following changes for steps : 

1. Create the application: summarizes the deployment, the type, and the size, with relevant links if needed
2. Connected Services
3.  Inject environment variables

Reply:
> Both Console-focused and CLI-focused instructions should be able to fit into guides, and users will choose whichever option suits them better.

We agree on this. So I think the better is to keep Console instructions, but simplify as you mentioned in Option 1 (and don't list a 7 steps process), focus on what's specific to Astro static deployment. And to Add a CLI section using existing instructions (including build cache stuff in both, maybe as an optional step). 

One point of attention on the Console instructions: as the whole creation process is being revamped, we'll have to update them after release. The more we detail it in each guide independently, parts of the doc, the more we'll have to make updates later.

For the SSR part discussed below, maybe we should just mention the user can use the Node adapter and send [to Astro doc](https://docs.astro.build/en/guides/integrations-guide/node/) about that in a dedicated section at the end, with instructions on what the user needs to do to activate it and the benefits.

I think one important message the reader should get when first seeing the page is: there is (almost) nothing special to do to deploy Astro on Clever Cloud. But if you want to go further (pnpm/yarn, build cache optimization, SSR), here's what you need to know.

Reply:
Settled for option 1 and SSR part being discussed above.